app/js/lib/form_validation.js

```diff
@@ -24,9 +24,9 @@ var jQuery = require( 'jquery' ),
 					return { status: 'OK' };
 			}
 
-			// Skip validation if not all required fields have been filled
+			// Return error status if not all required fields have been filled
 			if ( this.formValuesHaveEmptyRequiredFields( formValues, requiredFields ) ) {
-				return null;
+				return { status: 'ERR' };
```

Comment:
The tri-state return (null, object with status ERR and object with status OK) was deliberate (see https://github.com/wmde/FundraisingFrontend/blob/master/app/js/lib/redux_validation.js#L41-L43 ). 
Let's discuss the different approaches and their pros and cons in person.